### ISSUE TITLE
Ch4: misuse of term "learning goals"

### ISSUE THREAD

**Issue by labarba** [jupyter4edu/jupyter-edu-book#90]
The term "learning goals" is used as one feature heading for all pedagogical patterns, but what is described under that heading is _not_ learning goals. The heading could change from **Learning goals** to **Usage**.

Example:
> **4.2 Shift-Enter for the win** […] **Learning goals** This pattern can be used to introduce a topic or promote awareness about a set of tools.

Clearly not learning goals, but usage. This repeats in all case, except **4.18**, which reads: 
>Ability to translate from one field/language to another. Explain complex topics to someone from a different field.

That is, indeed, speaking about what the learner might be able to do after the exercise. None of the other cases do. In general, I don't think we can talk about "learning goals" of a pedagogical pattern; the learning objectives come when embedding that pattern in a subject matter.

**Comment by rwest** [comment]
While you're editing headings there.... 
I feel that "Format (lecture / lab / …)" would be better as "Format".  The extra was helpful in guiding authors using the template, but now adds nothing for the reader except words that may not apply (if it is a lecture or lab that will be clear when they read the following sentence, and if it's not they'll be confused).

**Comment by moorepants** [comment]
Would it be better to actually write learning goals for each one? It would seem preferable to me. My hunch would be that the original authors intended to write goals/objectives but there wasn't enough guidance on how to do so, thus the current results.

**Comment by labarba** [comment]
I do not think you can have _Learning goals_ on a pedagogical pattern. You get a learning goal when you design a lesson that applies the pedagogical pattern in a given context.

**Comment by moorepants** [comment]
I think that different pedagogical patterns can align better with some learning goals/objectives than others. For example, for the learning objective:

Student will be able to design and execute a for loop.

the pedagogical pattern "Fill in the blanks" works quite well for an objective like this, whereas, "Shift-Enter for the win" would like not be well aligned to that objective.

I was under the impression that the "learning goal" heading would give examples of learning objectives that work well with that pattern.
